uktrade/data-hub-frontend
# Rebuild edit company export countries form in React

## Description of change

As the title says

## Test instructions

Go to a company export tab and click "Edit export countries" the Typeahead should now be the React one

## Screenshots
### Before
<img width="790" alt="Screenshot 2020-03-26 at 13 42 41" src="https://user-images.githubusercontent.com/1481883/77653585-d1c6c200-6f67-11ea-8674-befc9d9e9be5.png">

### After
<img width="858" alt="Screenshot 2020-03-26 at 13 43 15" src="https://user-images.githubusercontent.com/1481883/77653628-e1dea180-6f67-11ea-9f5f-a2962f7b51bf.png">


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
